packages/mystmd-py/pyproject.toml

```diff
@@ -20,6 +20,10 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
     "Topic :: Scientific/Engineering",
 ]
+dependencies = [
+  "platformdirs~=4.2.2",
+  "nodeenv~=1.9.1"
```

Comment:
How does this play with conda? Do we have other node deps over there that do the install for us?

Reply:
Oh right, that's a good point. I didn't think about how we would repackage that. Let me have a think.